edx_sga/static/js/src/edx_sga.js

```diff
@@ -1,27 +1,40 @@
 /* Javascript for StaffGradedAssignmentXBlock. */
-function StaffGradedAssignmentXBlock(runtime, element) {
+function StaffGradedAssignmentXBlock(runtime, element, backendData) {
```

Comment:
Call this `options` or something more generic, other than backendData. You named it to describe where it comes from, but not what it's expected to have or do. `runtime` and `element` are generic, but descriptive enough if you're familiar with the concept of an XBlock.